owrx/source/perseussdr.py

```diff
@@ -0,0 +1,31 @@
+from .direct import DirectSource
+from owrx.command import Flag, Option
+
+
+#
+# perseustest -s 768000 -u 0 -f 14150000 -r-|csdr convert_s8_f|nmux --bufsize 192512 --bufcnt 260 --port 35989 --address 127.0.0.1
+# perseustest -a -t0    -o -
+# perseustest -d 9 -a -t 100000 -o -  -s 768000 -u 0 -f 14150000 -r-
```

Comment:
Can we get remove these? I don't think they're very helpful without explanation.